choco/nunit-console-with-extensions.nuspec

```diff
@@ -33,5 +33,15 @@
 	<language>en-US</language>
     <tags>nunit console runner test testing tdd</tags>
     <copyright>Copyright (c) 2017 Charlie Poole, Rob Prouse</copyright>
+    <dependencies>
+      <group>
+          <dependency id="nunit-console-runner" version="$version$" />
+          <dependency id="nunit-extension-nunit-project-loader" version="3.5.0" />
+          <dependency id="nunit-extension-vs-project-loader" version="3.5.0" />
+          <dependency id="nunit-extension-nunit-v2-result-writer" version="3.5.0" />
+          <dependency id="nunit-extension-nunit-v2-driver" version="3.6.0" />
+        <dependency id="nunit-extension-teamcity-event-listener" version="1.0.2" />
```

Comment:
>  Those listed are the last releases of each extension. 

I think @mikkelbu's right - there are later releases of these extensions already available on choco. Unless you were intending this to match the 3.7.1 release of the NUnit.Console NuGet package?